packages/reputation-miner/patricia-test.js

```diff
@@ -51,7 +51,7 @@ contract("Javascript Patricia Tree", accounts => {
       const fido = web3Utils.fromAscii("fido");
 
       await jsClient.reputationTree.insert(dog, fido);
-      await solClient.reputationTree.insert(dog, fido);
+      await solClient.reputationTree.insert(dog, fido, { gasLimit: 4000000 });
```

Comment:
This is a good reason to move these tests to `parity` or even `geth`, also it makes more sense overall the patricia tree tests are run against one of the "real" clients.